### PR TITLE
Update readme with specup instructions

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

Can’t assign you as reviewers, but please take a look @domwoe @l-wegner 

Thanks!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
+- forking the repo
+- editing the appropriate markdown files in the `/spec` folder
+
+The specification is rendered from the markdown files to the `index.html` specification file in the `/docs` folder
+using [Spec-Up](https://github.com/decentralized-identity/spec-up). The full guidance for using SpecUp is in that documentation.
+The short version of the instructions for this repository is:
+
+- Install prerequisites: node and npm
+- Run `npm install` from the root of the repository
+- Run `npm run edit` from the root of the repository to render the document with live updates to the `docs\index.html`. You can see the updates as you make as you edit the markdown files. Open the rendered file in a browser and refresh to see the updates as you work.
```
Comment (l-wegner) [inline readme.md:18]: `docs/index.html`

File: readme.md [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+- You can also run `npm run render` to just generate the specification file once.
+
+When you create a PR to update the specification, please **DO NOT** include `docs/index.html`. A GitHub Action on merge of PRs automagically
+renders for the full specification (`docs/index.html`) to the `gh-pages` branch in the repo and the specification is
+published from there. If there is a way to .gitignore the `index.html` in the main branch but not in the `gh-pages` branch, please let us know.
```
Comment (l-wegner) [inline readme.md:23]: Does it matter, if the `docs/index.html` is in the `main` branch?
The github action on commit to `main`, checks out `main`, runs `npm run render` (regenerates `docs/index.html`) and pushes the result to `gh-pages`. On `gh-pages` there's always only the version generated from the pipeline, if nobody ever commits to `gh-pages`.

Comment (domwoe) [reply readme.md:23]: Can't test it unfortunately, but one approach could be to add `docs/` to .gitignore but force adding the folder in the GHA worklow `git add -f docs/`

File: readme.md (hunk 1 above)
Comment (swcurran) [reply readme.md]: Thanks - fixed.

File: readme.md (hunk 2 above)
Comment (swcurran) [reply readme.md:23]: The problem I forsee is that if multiple people include index.html in the PR, and we merge the PRs in random order, we’ll get merge conflicts that will have to be resolved on that file.

I like @domwoe ’s idea — that sounds promising.  I think if we just add it do the GHA it should work.

@ryjones — any ideas from you on the question?

Comment (ryjones) [reply readme.md:23]: @swcurran I think that would work. You could test it on a fork and see if it does

**Pull request merged**